opentreemap/treemap/js/src/MapManager.js

```diff
@@ -8,7 +8,9 @@ var $ = require('jquery'),
     makeLayerFilterable = require('treemap/makeLayerFilterable'),
     urlState = require('treemap/urlState'),
 
-    _ZOOM_OPTIONS = {maxZoom: 21};
+    MAX_ZOOM_OPTION = {maxZoom: 21},
```

Comment:
Personally I've never seen a need to prefix private JS variables with `_`.  Since only explicitly exported items can be used anyway, it isn't necessary to distinguish public/private items (unlike Python, where everything is exported).